src/main/java/de/teamlapen/vampirism/world/loot/LootHandler.java

```diff
@@ -20,13 +21,6 @@
 public class LootHandler {
 
     public static final ResourceLocation STRUCTURE_VAMPIRE_DUNGEON = register("vampire_dungeon");
-    public static final ResourceLocation STRUCTURE_VILLAGE_TRAINER = register("village_trainer");
-    public static final ResourceLocation BASIC_VAMPIRE = register("entities/basic_vampire");
```

Comment:
I assume this is not necessary anymore?

Reply:
Could we avoid duplicate (imob and non imob) loot tables if we kept this?

Reply:
ok, reimplement it for imob and non imob entities

Reply:
see ModLootTables

in the next time i will try to move the stuff from LootHandler to ModLootTables